yjit/src/codegen.rs

```diff
@@ -746,96 +751,88 @@ pub fn gen_single_block(blockref: &BlockRef, ec: EcPtr, cb: &mut CodeBlock, ocb:
     block.set_start_addr(cb.get_write_ptr());
 
     // For each instruction to compile
+    // NOTE: could rewrite this loop with a std::iter::Iterator
     while insn_idx < iseq_size {
-
         // Get the current pc and opcode
-        //let pc = iseq_pc_at_idx(iseq, insn_idx);
-        //let opcode = opcode_at_pc(iseq, pc);
-        //assert!(opcode >= 0 && opcode < VM_INSTRUCTION_SIZE);
+        let pc = unsafe { rb_iseq_pc_at_idx(iseq, insn_idx) };
+        // try_into() call below is unfortunate. Maybe pick i32 instead of usize for opcodes.
```

Comment:
Yeah, I feel like our integer size stuff is all really awkward right now :-( We may need to wrap a fair bit of the C stuff, just to get a little more uniformity.